lib/watir/rails/middleware.rb

```diff
@@ -2,25 +2,49 @@ module Watir
   class Rails
     # @private
     class Middleware
+      class Counter
```

Comment:
I know that this is internal design implemenation here, but maybe use some more descriptive name (e.g. `PendingRequestCounter`) instead of a too generic `Counter`?